src/__tests__/components/NHSNotifyButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import { NHSNotifyButton } from '@/src/components/atoms/NHSNotifyButton/NHSNotifyButton';
+import { ButtonType } from '@/src/components/atoms/NHSNotifyButton/button.types';
+
+const buttonProps: ButtonType = {
+  children: 'Button text',
+};
+
+describe('Footer component', () => {
+  it('renders component correctly as a button', () => {
+    render(<NHSNotifyButton {...buttonProps} />);
+
+    expect(screen.getByTestId('link-button')).toBeInTheDocument();
+    expect(screen.getByTestId('link-button')).toHaveTextContent(
+      buttonProps.children?.toString() ?? 'Button text'
```

Comment:
given that buttonProps is a hardcoded value, the null coalesce is a bit unnecessary - you can just shorten it to either `buttonProps.children` or `'Button text'`